index.Rmd

```diff
@@ -1,105 +1,594 @@
+                "B. microti","B. miyamotoi","B. burgdorferi","A. phagocytophilum","B. microti","B. miyamotoi","B. burgdorferi")
+asdf$Life_Stage=c("Nymph","Nymph","Nymph","Nymph","Nymph","Nymph",
+                  "Adult","Adult","Adult","Adult","Adult","Adult")
+
+asdf2=tidyr::expand(asdf,Year,County,Pathogen,Life_Stage)
+asdf2$Total_tested=0
+asdf2$Tot_pos=0
+asdf2$percpos=0
+names(All_county_groupings)[4]="Total_tested"
+names(All_county_groupings)[5]="Tot_pos"
+names(All_county_groupings)[7]="Life_Stage"
+All_county_groupings2=rbind(All_county_groupings,asdf2)
+All_county_groupings3=tidyr::expand(All_county_groupings2,Year,Pathogen,County,Life_Stage)
+
+All_county_groupings4=left_join(All_county_groupings3,All_county_groupings,by=c("Year","Pathogen","Life_Stage","County"))
+names(test_copy)[4]="Life_Stage"
+
+
+
+test1 = All_county_groupings4 %>%
+        nest(-County)
+
+test2 = test1 %>% mutate(plot = map2(data, County, 
+                             ~ggplot()+ 
+                                     geom_hline(yintercept = 0,linetype='solid',color='black',size=.5,alpha=.7)+
+                                     geom_vline(xintercept = 2008,linetype='solid',color='black',size=.5,alpha=.7)+
+                                     geom_line(data=.x,aes(x=Year,y=percpos,color=Pathogen))+
+                                     geom_point(data=.x,aes(x=Year,y=percpos,color=Pathogen))+ scale_color_manual(name="Pathogen",values=c("#E41A1C","#377EB8","#4DAF4A","#984EA3","#E41A1C","#377EB8","#4DAF4A","#984EA3"))+
+                                     geom_line(data=test_copy,aes(x=Year,y=Avg_pos,color=Pathogen,linetype="NYS Average Pathogen Prevalence"))+
+                                     scale_linetype_manual("",values = c(2,2,2,2,1,1,1,1))+
+                                     theme(axis.text.x = element_text(angle=65,vjust=.65))+
+                                     facet_wrap(~Life_Stage)+
+                                     scale_x_continuous(breaks=c(2008:2019))+
+                                     ylab("Pathogen Prevalence (%)")+
+                                     labs(caption="Solid lines indicate pathogen prevalence from sample specimens in County")+
+                                     ggtitle(glue("{.y}"))))
+```
 
-Here's my first code chunk.
-```{r}
-1+2
+The third and final data source is land use and land cover data from the 2016 National Land Cover Dataset, available at the following link: [NLCD](https://www.mrlc.gov/data/nlcd-2016-land-cover-conus). These data were downloaded using R-specific NLCD calls. The call was looped to iterate the data download for each county, which would then overwrite the previous county's data to conserve storage. The analysis was also grouped into this code, which will be explained later.
+```{r NLCD_data,results='hide',error=F,message=F,warning=F,cache=TRUE}
+
+
+newcounties=County_Boundaries %>% filter(County != "New York",
```

Comment:
can also create a vector of names `counties=c("county1","county2")` and then `filter(county%in%counties)`